Sources/Networking/HTTPClient/HTTPClient.swift

```diff
@@ -63,9 +63,17 @@ class HTTPClient {
 extension HTTPClient {
 
     static func authorizationHeader(withAPIKey apiKey: String) -> RequestHeaders {
-        return ["Authorization": "Bearer \(apiKey)"]
+        return [Self.authorizationHeaderName: "Bearer \(apiKey)"]
     }
 
+    static func signatureUUIDHeader(with uuid: UUID) -> RequestHeaders {
+        return [Self.signatureHeaderName: uuid.uuidString]
+    }
+
+    static let authorizationHeaderName = "Authorization"
+    // TODO: confirm this name
+    static let signatureHeaderName = "X-Request-UIUD"
```

Comment:
typo `UIUD` -> `UUID`

Reply:
Well good thing I waited to confirm the name 🤣